index.js

```diff
@@ -101,30 +180,45 @@ module.exports = function(babel) {
         }
 
         // use "styled" as new default import, only if there's no such variable in use yet
```

Comment:
Ah, this comment is not up to date anymore. I'll fix it

Reply:
@danielberndt looking forward to it